src/site/xdoc/server/config-rabbitmq.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.    
+-->
+<document>
+
+ <properties>
+  <title>Apache James Server 3 - RabbitMQ Configuration</title>
+ </properties>
+
+<body>
+
+  <section name="RabbitMQ Server Configuration">
+      <p>Consult
+          <a href="https://github.com/apache/james-project/blob/master/dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/rabbitmq.properties">
+          rabbitmq.properties</a>
+          in GIT to get some examples and hints.
+      </p>
+
+      <p>
+          RabbitMQ Configuration
+      </p>
+      <dl>
+          <dt><strong>uri</strong></dt>
+          <dd>
+              the amqp URI pointing to RabbitMQ server. Details about amqp URI format is in <a href="https://www.rabbitmq.com/uri-spec.html">RabbitMQ URI Specification</a>
+          </dd>
+
+          <dt><strong>management.uri</strong></dt>
+          <dd>
+              the URI pointing to RabbitMQ Management Service. James need to retrieve some information about listing queues from this service in runtime.
+              Details about URI format is in <a href="https://www.rabbitmq.com/management.html#usage-ui">RabbitMQ Management URI</a>
+          </dd>
+
+          <dt><strong>management.user</strong></dt>
+          <dd>username to access management service</dd>
+
+          <dt><strong>management.password</strong></dt>
+          <dd>password to access management service</dd>
+      </dl>
+  </section>
+
+  <section name="RabbitMQ MailQueue Configuration">
+      <p>
+          RabbitMQ MailQueue Configuration
+      </p>
+      <p>
+          James mail queue is a component acting like a queue where it can enqueue and dequeue mails.
+          Beside of the basic features, it also allows some extra operations like getting size, browsing all items in the mail queue...
+          One of the mail queue implementation is RabbitMQ MailQueue, to able to provide those extra operations - which are the capabilities RabbitMQ can not offer,
```

Comment:
> One of the mail queue implementation is RabbitMQ MailQueue, to able to provide those extra operations - which are the capabilities RabbitMQ can not offer, the RabbitMQ MailQueue actually relies on both of RabbitMQ and Cassandra.

One of the mailqueue implementation is using RabbitMQ. As RabbitMQ doesn't offer enough features to implement efficiently all mailqueue operations, this implementation relies on Cassandra.